### PR TITLE
website/docs: new PR for the Entra provider docs (ignore old one)

### PR DESCRIPTION
**Pull request by tanberry**
This PR adds three new pages, for the new Entra ID provider, and adds Entra to the `sidebar.js` file.

Ignore PR #9687 it was too stale and got too messy when trying to rebase.

-   [x] The documentation has been updated
-   [x] The documentation has been formatted (`make website`)

### CODE REVIEW

**Review by BeryJu — APPROVED** [review]

File: website/docs/providers/entra/add-entra-provider.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+For more information about using a Entra ID provider, see the [Overview](./index.md) documentation.
+
+:::info
+This feature is in technical preview, so please report any bugs on [GitHub](https://github.com/goauthentik/authentik/issues).
+:::
+
+## Prerequisites
+
+To create a Entra ID provider provider in authentik, you must have already [configured Entra ID](./setup-entra.md) to integrate with authentik. You will need to obtain from Entra the Application (client) ID, the Directory (tenant) ID, and the Client secret values, to enter into authentik when adding an Entra ID provider.
```
Comment (kensternberg-authentik) [inline website/docs/providers/entra/add-entra-provider.md:17]: Copypasta typo: "To create an..."

Comment (kensternberg-authentik) [inline website/docs/providers/entra/add-entra-provider.md:17]: Typo? "You will need to obtain from Entra ID..."

File: website/docs/providers/entra/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+This discovery happens every time before a full sync is started.
+
+### Synchronization
+
+There are two types of synchronization: a direct sync and a full sync.
+
+A _direct sync_ happens when a user or group is created, updated or deleted in authentik, or when a user is added to or removed from a group. When one of these events happens, the direct sync automatically forwards those changes to Entra ID.
+
+The _full sync_ happens when the provider is initially created and when it is saved. The full sync goes through all users and groups matching the **User filtering** options set and will create/update them in Entra ID. After the initial sync, authentik will run a full sync every four hours to ensure the consistency of users and groups.
```
Comment (kensternberg-authentik) [inline website/docs/providers/entra/index.md:34]: "Every four hours..." Maybe we should say "This schedule cannot be changed from the configuration" or something.

**Review by kensternberg-authentik — APPROVED** [review]
Found some minor typos.

File: website/docs/providers/entra/add-entra-provider.md (hunk 1 above)
Comment (tanberry) [reply website/docs/providers/entra/add-entra-provider.md:17]: No typo but I reworded it to be less confusing.

Comment (tanberry) [reply website/docs/providers/entra/add-entra-provider.md:17]: fixed

**Comment by MaBauMeBad** [comment]
Just my 2 cents, but i think the naming should be matching.
What i mean with that is, as Provider you Name it Entra. As Source its Azure AD.
But its the same and therefore should be called the same to avoid irritation.

**Pull request merged**